package.json

```diff
@@ -13,6 +13,7 @@
     "start:frontend": "cd packages/frontend && yarn start"
   },
   "devDependencies": {
+    "aws-cdk": "^2.173.2",
```

Comment:
The "aws-cdk" dependency is also not required in root package.json. It's present in `infra/package.json`
```suggestion
```